tests/ref.txt

```diff
@@ -0,0 +1,124 @@
+    "cmip6:grid": "ORCA1 tripolar grid, 1 deg with refinement to 1/3 deg within 20 degrees of the equator; 361 x 290 longitude/latitude; 45 vertical levels; top grid cell 0-6.19 m",
+    "cmip6:mip_era": "CMIP6"
+  },
+  "geometry": {
+    "type": "Polygon",
+    "coordinates": [
+      [
+        [
+          0.049800001084804535,
+          -78.39350128173828
+        ],
+        [
+          0.049800001084804535,
+          89.74176788330078
+        ],
+        [
+          359.99493408203125,
+          89.74176788330078
+        ],
+        [
+          359.99493408203125,
+          -78.39350128173828
+        ],
+        [
+          0.049800001084804535,
+          -78.39350128173828
+        ]
+      ]
+    ]
+  },
+  "links": [
+    {
+      "rel": "source",
+      "href": "https://pavics.ouranos.ca/twitcher/ows/proxy/thredds/fileServer/birdhouse/testdata/xclim/cmip6/sic_SImon_CCCma-CanESM5_ssp245_r13i1p2f1_2020.nc",
+      "type": "application/x-netcdf",
+      "title": "birdhouse/testdata/xclim/cmip6/sic_SImon_CCCma-CanESM5_ssp245_r13i1p2f1_2020.nc"
+    }
+  ],
+  "assets": {
+    "HTTPServer": {
+      "href": "https://pavics.ouranos.ca/twitcher/ows/proxy/thredds/fileServer/birdhouse/testdata/xclim/cmip6/sic_SImon_CCCma-CanESM5_ssp245_r13i1p2f1_2020.nc",
+      "type": "application/x-netcdf",
+      "roles": [
+        "data"
+      ]
+    },
+    "OPENDAP": {
+      "href": "https://pavics.ouranos.ca/twitcher/ows/proxy/thredds/dodsC/birdhouse/testdata/xclim/cmip6/sic_SImon_CCCma-CanESM5_ssp245_r13i1p2f1_2020.nc",
+      "type": "text/html",
+      "roles": [
+        "data"
+      ]
+    },
+    "WCS": {
+      "href": "https://pavics.ouranos.ca/twitcher/ows/proxy/thredds/wcs/birdhouse/testdata/xclim/cmip6/sic_SImon_CCCma-CanESM5_ssp245_r13i1p2f1_2020.nc?service=WCS&version=1.0.0&request=GetCapabilities",
+      "type": "application/xml",
+      "roles": [
+        "data"
+      ]
+    },
+    "WMS": {
+      "href": "https://pavics.ouranos.ca/twitcher/ows/proxy/thredds/wms/birdhouse/testdata/xclim/cmip6/sic_SImon_CCCma-CanESM5_ssp245_r13i1p2f1_2020.nc?service=WMS&version=1.3.0&request=GetCapabilities",
+      "type": "application/xml",
+      "roles": [
+        "visual"
+      ]
+    },
+    "NetcdfSubset": {
+      "href": "https://pavics.ouranos.ca/twitcher/ows/proxy/thredds/ncss/birdhouse/testdata/xclim/cmip6/sic_SImon_CCCma-CanESM5_ssp245_r13i1p2f1_2020.nc/dataset.html",
+      "type": "application/x-netcdf",
+      "roles": [
+        "data"
+      ]
+    }
+  },
+  "bbox": [
+    0.049800001084804535,
+    -78.39350128173828,
+    359.99493408203125,
+    89.74176788330078
+  ],
+  "stac_extensions": []
```

Comment:
Still missing the `cmip6` extension.

Reply:
We are not using cmip6 extension. At least, in the UofT implementation, I am not going to use cmip6 extension.

Reply:
I don't understand? 
This is where all the `cmip6:` prefixed attributes are coming from. 
This is how STAC Items represent extensions and their corresponding schemas for validation. 

Using `cmip6:` without the extension essentially violates how STAC Items are represented.

Reply:
What exactly is stoping us from using https://github.com/TomAugspurger/cmip6?
I see a very strong overlap with all proposed properties.

Also, the https://github.com/stac-extensions/cf extension should be considered as well for any other parameter that would not be supported by the `cmip6` extension.

Reply:
I've already written about this above. I would recommend taking a look at those arguments above. Adding extensions, when not strictly necessary, adds complexity that makes the implementation a bad example to others for how to "easily" add items to STAC. There is nothing preventing another implementation for using an extension, I simply want to have an example that does not need to use extensions for a user in future who might not want to use extensions. 

> Using cmip6: without the extension essentially violates how STAC Items are represented.

Not sure what you mean here. As far as I can tell, there is nothing that *requires* properties with a prefix style to have associated extension. I've seen APIs that do not. Are you saying that as far as you know, doing this breaks something?

Reply:
It is part of the recommendations of STAC specification to use existing extensions for reusability and documentation. It is the main purpose of STAC to make content accessible in a standardized manner. Since both `cmip6` and `cf` extensions exist, they should be used instead of custom definitions. Using custom properties without any referenced extension and undefined by any JSON-spec that can validate them does not make it easier for users/web-clients to find the attributes, it complicates it on the STAC API side. STAC are even working with OGC to align redundant standards and reduce custom implementations essentially documentating the same data. Using un-prefixed properties will not "break" requirements, but it will strongly go against the STAC phylosophy. It is also better to guide users in employing recommended "best practices" instead of using workarounds. 

I would like to stress that this implementation is not only for UoT or a demo showcase. This exact implementation would be used by CRIM and Ouranosn to integrate it in actual instances. Therefore, I would prefer that we all streamline the STAC generation approach with a common strategy across all organizations.